src/Phinx/Console/Command/Create.php

```diff
@@ -52,6 +58,13 @@ protected function configure()
                 PHP_EOL,
                 PHP_EOL
             ));
+
+        // An alternative template.
+        $this->addOption('template', 't', InputOption::VALUE_REQUIRED, 'Use an alternative template');
+
+        // A classname to be used to gain access to the template content as well as the ability to
+        // have a callback once the migration file has been created.
+        $this->addOption('class', 'l', InputOption::VALUE_REQUIRED, 'Use a class implementing "' . self::CREATION_INTERFACE . '" to generate the template');
```

Comment:
Hmm.. http://php.net/manual/en/language.oop5.constants.php#114143 self:: vs static::, I assume that no matter what extension to the Create class exists, the value assigned to Create::CREATION_INTERFACE will always be used. Of course, hardcoding it is an option.